src/main/scala/fr/polytechnique/cmap/cnam/etl/events/BiologyAct.scala

```diff
@@ -0,0 +1,40 @@
+package fr.polytechnique.cmap.cnam.etl.events
+
+import java.sql.Timestamp
+import org.apache.spark.sql.Row
+
+trait BiologyAct extends AnyEvent with EventBuilder {
+
+  override val category: EventCategory[BiologyAct]
+
+  def fromRow(
```

Comment:
Do you really need this method?

Reply:
You are right. I removed this class and included DcirBiologyAct object in `MedicalAct.scala`. It does not change anything to the test. I think that this BiologyAct class was the cause of the bad coverage because I forgot to implement a test for this class.